web/views.py

```
@@ -6929,3 +6933,207 @@ def delete_post(request, post_id):
+        
+        # Combine date and time into a datetime string and make it timezone-aware
+        start_time = timezone.make_aware(timezone.datetime.fromisoformat(f"{date}T{time}"))
+        end_time = start_time + timezone.timedelta(minutes=int(duration))
+        
+        session = StudySession.objects.create(
+            study_plan=study_plan,
+            title=title,
+            description=description,
+            subject=subject,
+            start_time=start_time,
+            end_time=end_time
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)